tethysext/atcore/urls/app_users.py

```diff
@@ -209,7 +221,6 @@ def urls(url_map_maker, app, persistent_store_name, base_url_path='', base_templ
                 _persistent_store_name=persistent_store_name,
                 _AppUser=_AppUser,
                 _Organization=_Organization,
-                _Resource=_Resource,
```

Comment:
Not sure why the resource just went away here instead of becoming plural so pointing it out

Reply:
Turns out that view doesn't need the _Resource model.